recipes/ablog/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ablog" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "4027fbf75826bac4ad413eb5c8ab8e21bfe8e8c2635a07d9326873d9f14a27b7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - ablog = ablog.commands:ablog_main
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - werkzeug
+    - sphinx
+    - alabaster
+    - invoke
+    - python-dateutil
+    - sphinx-automodapi
+
+test:
+  source_files:
+    - docs
```

Comment:
This won't work, I just noticed,  since we don't put the docs in the PyPI tarball. 

I think we need to do the git clone as a testing command instead. 

`
- git clone --depth 1 git@github.com:sunpy/ablog.git
- cd ablog/docs
`